src/runners/tests/run_alerts.py

```diff
@@ -212,7 +212,7 @@ def test_alert_runners_processor_and_dispatcher(sample_alert_rules, update_jira_
     assert len(query_rule_run_record) == 7  # 3 from samples + 4 test alert queries
 
     assert query_rule_run_record[0]['QUERY_NAME'] == 'ACTIVITY_BY_ADMIN_ALERT_QUERY'
-    assert query_rule_run_record[0]['NUM_ALERTS_CREATED'] == 47
+    assert query_rule_run_record[0]['NUM_ALERTS_CREATED'] == 48
```

Comment:
this is not the only place where this count needs to increment, although it would help a lot to make it a variable that you can bump up in just one place